lib/smart_proxy_pulp_plugin/pulp_api.rb

```diff
@@ -17,5 +18,17 @@ class Api < Sinatra::Base
         log_halt 503, "Pulp server '#{URI.parse(::PulpProxy::Plugin.settings.pulp_url.to_s).host}' is unknown"
       end
     end
+
+    get '/status/disk_usage' do
+      size = (params[:size] && DiskUsage::SIZE.keys.include?(params[:size].to_sym)) ? params[:size].to_sym : :kilobyte
+      monitor_dirs = ::PulpProxy::Plugin.settings.to_h.delete_if { |key, _| key == :pulp_url || key == :enabled }
```

Comment:
Not great, as this will have to change the moment an additional setting is added; Why not explicitly list settings you are interested in?